packages/apps/docs/src/docs/deploy/chainweb/chainweb-get-started.md

```diff
@@ -22,13 +25,487 @@ Before installing Chainweb, you should verify that your computer meets the follo
 - Network: Publicly-accessible IP address.
 - Operating system: Linux AMD64 architecture.
 
-If you also plan to use the node as an API server for Pact or for mining, Rosetta API integration, or accessing indexed data using `chainweb-data` you should verify that your computer meets the following minimum recommended hardware requirements: 
+If you also plan to use the node as an API server for Pact calls or for mining, exchange integration, or accessing indexed data, you should verify that your computer meets the following minimum recommended hardware requirements: 
 
 - Four (4) CPU cores.
 - Eight (8) GB of RAM.
 
+### Operating system architecture
+
+It's possible to run a Chainweb node on Linux or macOS with ARM64 architecture.
+However, you must build the binaries yourself and you might experience some unexpected behavior when running a node on ARM64 architecture.
+If you want to run a Chainweb node on Microsoft Windows, you should install the latest Windows Services for Linux (WSL) software.
```

Comment:
this is possible, but the Chainweb team doesn't really want to provide support to anyone experiencing issues on WSL. maybe add a note about that?